api-spec/STAC-extensions.yaml

```diff
@@ -548,24 +547,14 @@ components:
         default: 10
       style: form
       explode: false
-    next:
-      name: next
-      in: query
-      description: >-
-        The token to retrieve the next set of results, e.g., offset, page,
-        continuation token
-      required: false
-      schema:
-        $ref: '#/components/schemas/next'
-      style: form
     ids:
       name: ids
       in: query
       description: >
         Array of Item ids to return. All other filter parameters that further
         restrict the number of 
 
-        search results (except `next` and `limit`) are ignored
+        search results are ignored
```

Comment:
I think `limit` is still ignored, right?

Reply:
Do we support paging when using ids? It seems a little strange as the user is requesting the exact items they want.